src/test/java/com/example/weather/client/mappers/WeatherDataMapperTest.java

```diff
@@ -7,11 +7,13 @@
 import org.junit.jupiter.api.Test;
 
 import static com.example.weather.client.utility.PodamUtility.makePojo;
-import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
 
 class WeatherDataMapperTest {
 
-    private WeatherDataDto weatherDataDto = makePojo(WeatherDataDto.class);
+    private WeatherDataDto expectedWeatherDataDto = makePojo(WeatherDataDto.class);
+    private WeatherData expectedWeatherData = makePojo(WeatherData.class);
     private final WeatherDataMapper weatherDataMapper = new WeatherDataMapper(
             new GeoCoordinatesMapper(),
             new WeatherMapper(),
```

Comment:
from what I see now, those components have dedicated tests, so they can be mocked and the results of their mappings don't need assertions here.